docs/release_notes.rst

```diff
@@ -1,23 +1,11 @@
 .. _release-notes:
 
-=============
-Release Notes
-=============
-
-This page contains the release notes for Qiskit, starting from Qiskit 0.45, the first time that Qiskit and Qiskit Terra had the same versions.
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+Qiskit |version| release notes
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
```

Comment:
Fwiw, the choice of punctuation doesn't affect what level of heading is generated, and we tend to use `=====` under/over line in the API docs to be the L1 heading (though I know we use `%%%%` under/over line in `index.rst`).  Not important either way, though.